charts/nginz/conf/static/zauth.acl

```diff
@@ -4,11 +4,13 @@ a (blacklist (path "/provider")
              (path "/bot/**")
              (path "/i/**"))
 
-u (whitelist (path "/access"))
```

Comment:
whitelisting /access for user tokens has no effect, as that route is not zauth-authenticated

Reply:
> whitelisting /access for user tokens has no effect, as that route is not zauth-authenticated

OK, I was wondering about this - Does this not mean though that this is the only route where user tokens are allowed/accepted?

Reply:
As far as I understand:

* User Tokens are always sent inside a cookie header. Access Tokens are always sent either as an `Authorization: Bearer <token>` or a query paramater.
* nginz does nothing with Cookies. Cookies are just passed through and remain untouched by nginz's zauth module.
* It just happens that the cookies generated in brig contain `path=/access`, meaning the browser only sends them to /access, not to other routes. 
* I don't know what mobile clients do. Perhaps they send cookies elsewhere, perhaps not, we don't know as we don't check for them.

For stuff in the `Authorization: ` header, nginz zauth looks at them: https://github.com/wireapp/wire-server/blob/develop/services/nginz/third_party/nginx-zauth-module/zauth_module.c#L303-L323 (if it doesn't find any authroization header it also looks for a query parameter, and if that is also not there it apparently looks for a `zprovider` cookie. But it doesn't look like the line I removed would have any effect. I admit I only have a partial understanding of zauth, so I might be wrong